requirements.txt

```diff
@@ -1,4 +1,4 @@
-django>=1.11,<2.2
+django>=1.11,<3.0
```

Comment:
don't use `3.0`, use `2.3`, same applies for the other occurrences

Reply:
@oltarzewskik :point_up:

Reply:
@nemesisdesign Ok, changed